attributes/default.rb

```diff
@@ -5,44 +5,20 @@
 
 default['qubit_bamboo']['log_path'] = '/var/log/bamboo-server.log'
 
-default['qubit_bamboo']['config']['marathon_port'] = 8080
-default['qubit_bamboo']['config']['marathon_hosts'] = ["http://localhost:#{node['qubit_bamboo']['config']['marathon_port']}"]
+default['qubit_bamboo']['marathon_port'] = 8080
+default['qubit_bamboo']['config']['Marathon']['Endpoint'] = ["http://localhost:#{node['qubit_bamboo']['marathon_port']}"].join(',')
 
 default['qubit_bamboo']['config']['port'] = 8000
```

Comment:
You will have to rename this value if you don't want it ending up in the generated JSON.